ecdsa_test.go

```diff
@@ -98,11 +98,12 @@ func TestI2OSP(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			err := I2OSP(tt.x, tt.buf)
-			if (err != nil) != tt.wantErr {
+			if err != nil && (err.Error() != tt.wantErr) {
```

Comment:
What if `err == nil` but `tt.wantErr` is non-empty?

Reply:
Done the changes in the recent commit